relay/config.go

```diff
@@ -0,0 +1,41 @@
+package relay
+
+import core "github.com/Layr-Labs/eigenda/core/v2"
+
+// Config is the configuration for the relay Server.
+type Config struct {
+
+	// Shards contains the IDs of the relays that this server is willing to serve data for. If empty, the server will
+	// serve data for any shard it can.
+	Shards []core.RelayKey
+
+	// MetadataCacheSize is the size of the metadata cache. Default is 1024 * 1024.
```

Comment:
Can we add some units to the config descriptions?

Reply:
Description added.

```
// MetadataCacheSize is the size of the metadata cache. Current cache implementation is unaware of data sizes, and 
	// so this is a total count, not a size in bytes. Default is 1024 * 1024.
	MetadataCacheSize int
```